# Microseconds since system boot
uint64 timestamp
float32  lambda_p_x
float32  lambda_p_y
float32  lambda_p_z
float32  lambda_q_x
float32  lambda_q_y
float32  lambda_q_z
float32  iusl_kq

float32  iusl_kw_x
float32  iusl_kw_y
float32  iusl_kw_z
float32  iusl_kv_x
float32  iusl_kv_y
float32  iusl_kv_z

float32  iusl_wa1
float32  iusl_wa2
float32  iusl_wa3

float32  iusl_pos_px
float32  iusl_pos_py
float32  iusl_pos_pz
float32  iusl_pos_ix
float32  iusl_pos_iy
float32  iusl_pos_iz
float32  iusl_pos_dx
float32  iusl_pos_dy
float32  iusl_pos_dz
